chia/consensus/block_body_validation.py

```diff
@@ -180,36 +180,25 @@ async def validate_block_body(
 
             # The generator_refs_root must be the hash of the concatenation of the List[uint32]
             generator_refs_hash = std_hash(
-                b"".join([(i).to_bytes(4, byteorder="big") for i in block.transactions_generator_ref_list])
+                b"".join([i.to_bytes(4, byteorder="big") for i in block.transactions_generator_ref_list])
```

Comment:
Maybe we should use `uint32` here rather than this ad-hoc serialization code.

Reply:
yeah good point